sdk/appconfiguration/app-configuration/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,78 @@
+
+For detailed instructions on how to enable logs, see the [@azure/logger package docs](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/core/logger).
+
+### Authentication issues
+
+In addtition to connection strings, Azure App Congifuration supports [role-based access control](https://learn.microsoft.com/azure/role-based-access-control/overview) (RBAC) using Azure Active Directory authentication. To provide a valid credential, you can use the `@azure/identity` dependency. For more details on getting started, see the [README](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/appconfiguration/app-configuration) of Azure App Configuration library. 
+
+#### Permission issues
+
+Calls to service clients resulting in an error with HTTP code of 401 or 403 often indicate the caller doesn't have sufficient permissions for the specified API. Check the service documentation to determine which RBAC roles are needed for the specific request, and ensure the authenticated user or service principal have been granted the appropriate roles on the resource. More information on App Configuration roles can be found [here](https://learn.microsoft.com/azure/azure-app-configuration/concept-enable-rbac#azure-built-in-roles-for-azure-app-configuration).
+
+For more help with troubleshooting authentication errors, see the Azure Identity client library [troubleshooting guide](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/TROUBLESHOOTING.md).
+
+### Limit Issues
+Azure App Configuration service methods throw a RestError or its subclass on failure. The error thrown by the App Configuration client library includes detailed response error object that provides specific useful insights into what went wrong and includes corrective actions to fix common issues. A common error encountered is HTTP status code 429 for exceeding limit. Refer to the body of the 429 response for the specific reason why the request failed. The failure often happen under these circumstances:
+
+* Exceeding the daily request limit for a store in the Free tier.
+* Exceeding the hourly request limit for a store in the standard tier.
+* Momentary throttling due to a large burst of requests†.
+* Excessive bandwidth usage.
+* Attempting to create or modify a key when the storage quota is exceeded.
+
+To reduce number of requests made to App Configuration service, please check out [this guide](https://learn.microsoft.com/azure/azure-app-configuration/howto-best-practices#reduce-requests-made-to-app-configuration).
+
+## Get additional help
+
+Additional information on ways to reach out for support can be found in the [FAQ](https://learn.microsoft.com/azure/azure-app-configuration/faq) for Azure App Configuration service and common best practice sample can be found in [Best Practice Samples](https://learn.microsoft.com/azure/azure-app-configuration/howto-best-practices).
```

Comment:
Could you also mention logging?

ref: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/formrecognizer/ai-form-recognizer/TROUBLESHOOTING.md#logging

Reply:
Is that the same as the "Logging using setLogLevel" section? Or adding it to the additional help section?

Reply:
same as the "Logging using setLogLevel" section